ElectricPokemon.java

```diff
@@ -0,0 +1,54 @@
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class ElectricPokemon extends Pokemon {
+    private final Map<String, Integer> attacks;
+    public ElectricPokemon(String name, int level, String... pokemonType) {
+        super(name, level, pokemonType);
+        attacks = new HashMap<>();
+        initializeAttacks();
+    }
+
+    private void initializeAttacks() {
+        attacks.put("thunder", 20);
+        attacks.put("thunderPunch", 15);
+        attacks.put("electroBall", 25);
+        attacks.put("thunderCage", 35);
+    }
+
+    public void eats(String foodName) {
+        super.eats(foodName);
+    }
+
+    public void useAttack(String attackName, Pokemon enemy) {
```

Comment:
Deze methode lijkt in elke subklasse hetzelfde te zijn. Misschien kun je deze dan beter in de superklasse definieren.